docs/SITECHANGELOG.md

```diff
@@ -0,0 +1,49 @@
+# MvvmCross site changelog
+
```

Comment:
Can we add a little text to the top paragraph? "This is the MvvmCross site changelog, changed to the website will be listed here". Or maybe we shouldn't have two changelogs at all and put this in our normal, is it "unnecessary" extra work? I think @Cheesebaron can form a better opinion on this